docs/design/coreclr/jit/viewing-jit-dumps.md

```diff
@@ -120,29 +120,31 @@ These can be set in one of three ways:
 
 ## Specifying method names
 
-The complete syntax for specifying a single method name (for a flag that takes a method name, such as `COMPlus_JitDump`) is:
+Some environment variables such as `DOTNET_JitDump` take a set of patterns specifying method names. The matching works in the following way:
```

Comment:
At some point (but not in this change) we should probably make a complete change of our documentation to use the DOTNET_ form instead of the COMPlus_ form, but it's odd to only change this one case in this file.

Reply:
Sure, I'll revert this.

```suggestion
Some environment variables such as `COMPlus_JitDump` take a set of patterns specifying method names. The matching works in the following way:
```